apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -81,6 +81,14 @@ export const StandardUsage: FunctionComponent = () => {
 
   tokens.backgroundColor = coinColor;
 
+  const fontBuiltInProps = {
+    fontFamily: 'Arial',
+    codepoint: 0x2663,
+    fontSize: 40,
+  };
+
+  const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
```

Comment:
Thanks for adding this check! Have you had a chance to boot the app in windows and make sure that it loads the page?

Reply:
Oh, I haven't added Avatar to windows yet. Thanks for catching it. I'll add it